docs/api/misc.md

```diff
@@ -73,3 +73,19 @@ int res0IndexCount();
 ```
 
 Number of resolution 0 **H3** indexes.
+
+## getPentagonsAtRes
+
+```
+void getPentagonsAtRes(int res, H3Index *out);
+```
+
+All the pentagon **H3** indexes at specified resolution.
```

Comment:
nit: "at the specified resolution."

Please add a note that `out` must be of size `pentagonIndexCount()`